pulp_service/pulp_service/app/settings.py

```diff
@@ -4,3 +4,7 @@
 .. _Plugin Writer's Guide:
     https://docs.pulpproject.org/pulpcore/plugins/plugin-writer/index.html
 """
+
+RH_IDENTITY_HEADER = "HTTP_X_RH_IDENTITY"
+SUBSCRIPTION_API_URL = "https://subscription.stage.api.redhat.com/svcrest/subscription/v5/featureStatus?accountId=5894300&features=RHEL-OS-x86_64&features=RHEL-EUS-x86_64"
```

Comment:
```suggestion
SUBSCRIPTION_API_URL = "https://subscription.stage.api.redhat.com/svcrest/subscription/v5/featureStatus"
```

You append this later I think right?